deps/outliner/src/logseq/outliner/tree.cljs

```diff
@@ -0,0 +1,135 @@
+(ns logseq.outliner.tree
```

Comment:
The outliner module has been moved to its own dep so that it can be used in both the UI thread and workers.